CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Change Log
 
+## [3.1.0](https://github.com/auth0/Auth0.swift/tree/3.1.0) (2026-07-29)
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)